setup-data-node.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+if [[ $EUID > 0 ]]; then  
+  echo "Please run as root/sudo"
+  exit 1
+fi
+
+HOME_DIR=$(eval echo ~${SUDO_USER:-${USER}})
+CURRENT_USER="${SUDO_USER:-${USER}}"
+DEFAULT_USER="wernerfamily"
+
+SCRIPT_URL="https://raw.githubusercontent.com/gauravpathak/humidity-logger/master/logger.py"
+SERVICE_URL="https://raw.githubusercontent.com/gauravpathak/humidity-logger/master/humidity-logger.service"
+CONFIG_URL="https://raw.githubusercontent.com/gauravpathak/humidity-logger/master/config.toml.example"
+
+GRAFANA_DATA_SRC_FILE="influxdb-datasource.yaml"
+GRAFANA_DATA_SRC_URL="https://raw.githubusercontent.com/gauravpathak/humidity-logger/setup-script/influxdb-datasource.yaml.example"
```

Comment:
The `username` and `branch` needs to be updated before merging.  
@benediktwerner Please let me know if you wish to merge these changes, it needs to be modified as per the username and active branch.